scripts/common.sh

```diff
@@ -193,16 +193,43 @@ function msg_on_error
     fi
 }
 
+
 function diff_values
 {
     (( $# >= 2 )) || fail "diff_values requires at least 2 arguments."
 
     local value1="$1"
     local value2="$2"
-    shift
-    shift
+    shift 2
 
-    diff --unified=0 <(echo "$value1") <(echo "$value2") "$@"
+    if ! diff --unified=0 <(echo "$value1") <(echo "$value2") "$@"
+    then
+        if [[ -n ${DIFFVIEW:-} ]]
+        then
+            # Use user supplied diff view binary
+            "$DIFFVIEW" "${value1}" "${value2}"
```

Comment:
Does this actually work? Since these are not files, you probably need this instead:
```suggestion
            "$DIFFVIEW" <(echo "$value1") <(echo "$value2")
```
But please test it locally.

Reply:
You're awesome!